bin/common.h

```diff
@@ -35,6 +35,22 @@
         }                         \
     } while (0)
 
+#define ENSURE_EXIT(x, msg)       \
+    do {                          \
+        if (!(x)) {               \
+            print_s2n_error(msg); \
```

Comment:
`print_s2n_error` prints s2n-tls error and debug messages, and it doesn't seem like ENSURE_EXIT/ENSURE_RETURN can be used in a way to make these messages relevant:
https://github.com/aws/s2n-tls/blob/3d125a514dfe0a9e4d620f22eb40703088ac53fb/bin/echo.c#L54-L57

Should these macros maybe just print the message without the s2n-tls error messages?